catalog/core/catalog-core-solrcommands/pom.xml

```diff
@@ -198,7 +204,8 @@
                             javax.xml.xpath,
                             org.apache.commons.codec.binary,
                             org.apache.commons.logging; version="[1.1.0,2.0.0)",
-                            org.apache.felix.gogo.commands,
+                            org.apache.karaf.shell.api.action,
```

Comment:
❔ Do we need to embed the karaf shell API across all commands? I assume a bundle isn't exporting those API or this is the best way to do it anyway, just confirming.